doc/getting_started/installation_linux.rst

```diff
@@ -150,6 +165,15 @@ Follow these steps to install the SDK on your Linux host system.
       export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
       export ZEPHYR_SDK_INSTALL_DIR=<sdk installation directory>
 
+   .. note::
+      On Linux distributions where the environment has default CFLAGS
```

Comment:
For clarity (since you don't mention which distros this applies to), I suspect you can always unset these variables, so we could just say here:

    Some Linux distributions have default CFLAGS and CXXFLAGS
    environment variables already set.  For all distros, they need to be
    unset to prevent these settings from interfering with cmake: